src/modules/commander/PreflightCheck.cpp

```diff
@@ -682,6 +682,35 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &vehicle_s
 	return success;
 }
 
+static bool failureDetectorCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &status, bool report_fail, bool prearm)
+{
+	bool success = true;
+
+	if (!prearm) {
+		// Ignore failure detector check after arming.
+		return true;
+
+	} else {
```

Comment:
The else is unnecessary, you can remove it.

Reply:
The `else` is still there, right?